core/rend/gui.cpp

```diff
@@ -774,95 +774,104 @@ static void controller_mapping_popup(std::shared_ptr<GamepadDevice> gamepad)
 	ImGui::PopStyleVar();
 }
 
+//ImGui DOES NOT support multiple non-stacked modal popup
+bool no_popup_opened()
+{
+    return !ImGui::IsPopupOpen(0, ImGuiPopupFlags_AnyPopupId);
+}
+
 static void error_popup()
 {
-	if (!error_msg.empty())
+    static auto popup_msg = error_msg;
+    if (!error_msg.empty() && no_popup_opened())
```

Comment:
This avoid clashing approach should be submitted in the original Flycast repo also, what should be the preferred practise?

Reply:
The original flycast code uses hard-tab, such changes cause conflicts.